tutorials/continuous-aggs-tutorial.md

```diff
@@ -220,38 +240,90 @@ max | 2016-01-31 23:59:59
 ```
 
 The column `job_id` gives the id of the background worker that updates the continuous
-aggregate query. `next_scheduled_run` says when the next scheduled update
-will occur.  The `completed_threshold` shows that rows with
-`pickup_time` value < '2016-01-31 22:00:00' (from the *rides* table) were used
-to update the continuous aggregate. Since the `refresh_lag` is set to 2 hours,
-the completed threshold is 2 hours behind the maximum pickup_time in the  
-*rides* table.  After a job completes, the `invalidation_threshold` and
-`completed_threshold` will be the same. These values differ when a background
-job is running.
+aggregate. `next_start` says when the next scheduled update will occur. 
+`last_sucessful_finish` and `last_run_duration` allow you to see when the
+scheduled aggregate finished and how long it took to run.
 
+### 5. Update the Continuous Aggregate schedule
+Altering the schedule or parameters of a continuous aggregate policy is accomplished by
+first removing the existing policy and then creating a new updated policy with the
+desired settings.
 
-### 5. Alter and Refresh of continuous aggregates
-The parameters passed in the `WITH` clause can be modified using `ALTER VIEW`.  
-We can modify the `refresh_lag` to 1 hour using `ALTER VIEW`.
+To modify the policy that currently exists on the `cagg_rides_view` so that the `start_offset`
+is now equal to `1 month`, you would:
 
 ``` sql
-ALTER VIEW cagg_rides_view SET (timescaledb.refresh_lag='1h');
-ALTER VIEW
-
- SELECT view_name, refresh_lag, refresh_interval, max_interval_per_job, materialization_hypertable
-FROM timescaledb_information.continuous_aggregates;
--[ RECORD 1 ]--------------+-------------------------------------------------
-view_name                  | cagg_rides_view
-refresh_lag                | 01:00:00
-refresh_interval           | 00:30:00
-max_interval_per_job       | 20:00:00
-materialization_hypertable | _timescaledb_internal._materialized_hypertable_2
+SELECT remove_continuous_aggregate_policy('cagg_rides_view');
+
+SELECT add_continuous_aggregate_policy('cagg_rides_view', 
+  start_offset => interval '1 month',
+  end_offset   => INTERVAL '1 hour',
+  schedule_interval => INTERVAL '30 minutes');
+
+SELECT view_name, schedule_interval, 
+  config ->> 'start_offset' as start_offset,
+  config ->> 'end_offset' as end_offset,
+  date_trunc('second',next_start::timestamp) as next_start,
+  materialization_hypertable_name
+FROM timescaledb_information.continuous_aggregates ca
+  INNER JOIN timescaledb_information.jobs j 
+    ON ca.materialization_hypertable_name = j.hypertable_name
+  WHERE view_name = 'cagg_rides_view';
+
+-[ RECORD 1 ]-------------------+---------------------------
+view_name                       | cagg_rides_view
+schedule_interval               | 00:30:00
+start_offset                    | 1 mon
```

Comment:
It's "1 mon" not "1 month"?

Reply:
yeah - surprised me too. Does that seem reasonable to others?